.github/workflows/build-test.yaml

```diff
@@ -1448,9 +1448,94 @@ jobs:
           echo "::notice ::support bundle uploaded to aws replicated-dev account s3://kots-e2e-build-test-support-bundles/${BUNDLE}"
 
 
+  validate-helm-install-order:
+    runs-on: ubuntu-20.04
+    needs: [ can-run-ci, build-kots, build-kurl-proxy, build-migrations, push-minio, push-postgres ]
+    strategy:
+      fail-fast: false
+      matrix:
+        k8s_version: [ v1.23.3-k3s1 ]
+    steps:
+      - uses: replicatedhq/action-k3s@main
+        id: k3s
+        with:
+          version: ${{ matrix.k8s_version }}
+
+      - name: download kots binary
+        uses: actions/download-artifact@v2
+        with:
+          name: kots
+          path: bin/
+
+      - run: chmod +x bin/kots
+
+      - name: run the test
+        env:
+          APP_SLUG: helm-install-order
+        run: |
+          set +e
+          echo ${{ secrets.HELM_INSTALL_ORDER_LICENSE }} | base64 -d > license.yaml
+          ./bin/kots \
+            install $APP_SLUG/automated \
+            --license-file license.yaml \
+            --no-port-forward \
+            --namespace $APP_SLUG \
+            --shared-password password \
+            --kotsadm-registry ttl.sh \
+            --kotsadm-namespace automated-${{ github.run_id }} \
+            --skip-preflights \
```

Comment:
how about removing the preflights from the app itself instead?

Reply:
see https://github.com/replicatedhq/kots-test-apps/tree/main/helm-install-order - I think it's stuck on the default preflights?

Reply:
huh, there shouldn't be such a thing as default preflights. ah i think this is a bug that's fixed in another pr that is not yet merged: https://github.com/replicatedhq/kots/pull/2763